### DISCUSSION TITLE
RDNSS MUST in the DNS section

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#35] [General]
Hi Brian,
> As noted in [2. Managed configuration](https://github.com/becarpenter/book6/blob/main/2.%20IPv6%20Basic%20Technology/Managed%20configuration.md), the DNS server for a subnet must be announced by a Router Advertisement even if DHCPv6 is in use.

I have not found it in the "Managed configuration" section and I do not understand it in general:
why RDNSS is the MUST?
Option 23 of DHCP is a perfectly fine competitor.

**Comment by timchown** [comment]
On 22 Nov 2022, at 14:00, Eduard Vasilenko ***@***.***> wrote:

Hi Brian,

As noted in 2. Managed configuration<https://github.com/becarpenter/book6/blob/main/2.%20IPv6%20Basic%20Technology/Managed%20configuration.md>, the DNS server for a subnet must be announced by a Router Advertisement even if DHCPv6 is in use.

I have not found it in the "Managed configuration" section and I do not understand it in general:
why RDNSS is the MUST?
Option 23 of DHCP is a perfectly fine competitor.

Support for it is a MUST in RFC8504 and mandatory in RIPE 772

Doesn’t mean you have to use it.

Tim

**Comment by becarpenter** [comment]
Yes, my mistake, will fix.

**Comment by becarpenter** [comment]
Closed as done
